pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,22 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+	var found = false
+	for _, logLevel := range override.LogLevel {
+		if logLevel.ContainerName == c.Name {
```

Comment:
Looks like there isn't any validation of these entries. We probably need some validation added up near the other reconcile validations to ensure the containerName is unique, one of the allowed names, and that the log level is within a valid range.

Reply:
the validation is done by the CRD validation in `+kubebuilder:validation:Pattern=^(reconciler|git-sync|hydration-controller|oci-sync|helm-sync)$`

Reply:
Do any of the components error if verbosity is less than 0?
Technically I don't think there's a max level in the tools we use, but we could maybe scan the code and check for `.V(\d+)` and see what the max is and pick that way... maybe just 0-10?
Do we want to allow disabling logs with -1?

Reply:
it looks like the values that are being used are from 0 to 9. I think restricting to 0 to 10 would be a good idea